packages/vuetify/src/components/VBtn/VBtn.tsx

```diff
@@ -155,23 +166,24 @@ export const VBtn = defineComponent({
             locationStyles.value,
             sizeStyles.value,
           ]}
-          disabled={ isDisabled.value || undefined }
-          href={ link.href.value }
+          disabled={isDisabled.value || undefined}
+          href={link.href.value}
```

Comment:
I've reset my VSCode settings to make sure I wasn't picking up any other configuration, and run separately the linter.

These fixes seem to be legit so the file was not formatted correctly previously.

Reply:
No it is correctly formatted. There should be spaces.

Reply:
You have prettier or something running